### PR TITLE
Added the docs for TBv10 Controls - Key Vault, Load balancer and SPN password credentials. 

### PR DESCRIPTION
**Pull request by jaiswalricha**

### CODE REVIEW

File: Control coverage/Feature/KeyVault.md [hunk 1]
```diff
@@ -236,4 +237,40 @@ properties.networkAcls<br />
...
+### Display Name
+Key Vaults must have Soft Delete and Purge Protection enabled
+
+### Rationale
+Malicious deletion of a key vault can lead to permanent data loss. A malicious insider in your organization can potentially delete and purge key vaults. Purge protection protects you from insider attacks by enforcing a mandatory retention period for soft deleted key vaults. No one inside your organization or Microsoft will be able to purge your key vaults during the soft delete retention period.
+
+### Control Spec
+
+> **Passed:**
+> if both Soft Delete and Purge Protections are enabled.
```
Comment (TarunKrShukla) [inline Control coverage/Feature/KeyVault.md:253]: I in 'if' should be caps.

File: Control coverage/Feature/KeyVault.md [hunk 2]
```diff
@@ -236,4 +237,40 @@ properties.networkAcls<br />
...
+>
+> **Failed:**
+> if Soft Delete or Purge Protections is disabled.
```
Comment (TarunKrShukla) [inline Control coverage/Feature/KeyVault.md:256]: I in 'if' should be caps.

File: Control coverage/Feature/LoadBalancer.md [hunk 3]
```diff
@@ -90,5 +92,36 @@ OR <br/>No Virtual Network found attached either with FrontEnd or Backend of Loa
...
+### Display Name 
+Azure Load Balancer with no backend pools should be removed
+
+### Rationale 
+Load balancer distributes inbound flows that arrive at the load balancer's front end to backend pool instances. If there are no backend pool(s), Load balancer is simply unused. Cleaning up unused Load balancer is suggested as a good hygiene practice.
+
+### Control Spec 
+
+> **Passed:** 
+> if Backend pools are attached with the Load balancer.
```
Comment (TarunKrShukla) [inline Control coverage/Feature/LoadBalancer.md:109]: same as above.

File: Control coverage/Feature/SubscriptionCore.md [hunk 4]
```diff
@@ -1072,3 +1073,47 @@ This Service depends mainly on 3rd party identity provider, and that can cause a
...
+
+        #To get the Service Principal/Enterprise Application's Password Credentials List. This list contains the Key Id and other details.
+        $passwordCredentials = Get-AzureADServicePrincipalPasswordCredential -ObjectId "Enterprise Application Object Id"
+	
+        #Provide the Key Id of the Password Credentials which you want to remove.
+        $spn = Remove-AzureADServicePrincipalPasswordCredential -ObjectId "Enterprise Application Object Id" -KeyId "Key Id of the credential"	
+	 ```  
+### Azure Policy or ARM API used for evaluation 
+
+- Graph API to fetch secret details: - /v1.0/servicePrincipals?$filter=id in ({ServicePrincipalObjectId})<br />
```
Comment (TarunKrShukla) [inline Control coverage/Feature/SubscriptionCore.md:1113]: Actually heading is confusing now, as it's not a ARM policy.

File: Control coverage/Feature/KeyVault.md (hunk 1 above)
Comment (jaiswalricha) [reply Control coverage/Feature/KeyVault.md]: done

**Review by TarunKrShukla — APPROVED** [review]

**Review by Aboli-msft — APPROVED** [review]

**Pull request merged**